src/frontend/src/catalog/index_catalog.rs

```diff
@@ -45,6 +48,11 @@ pub struct IndexCatalog {
 
     pub secondary_to_primary_mapping: BTreeMap<usize, usize>,
 
+    /// Map function call from the primary table to the index table.
+    #[derivative(PartialEq = "ignore")]
+    #[derivative(Hash = "ignore")]
```

Comment:
Why do we exclude this field from `Eq` and `Hash`? 👀 Is it since that it can be derived by `index_item`?

Reply:
Yes, it can be derived by `index_item`. Another reason is that `FunctionCall` can not be used as `BTreeMap`'s key.

Reply:
Get it. What about documenting it here?